app/pages/privacy-policy.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+import PrivacyPolicy from './privacy-policy';
+
+describe('PrivacyPolicy', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<PrivacyPolicy />);
+  });
+
+  it('renders without crashing', () => {
+    const privacyPolicyContainer = wrapper.find('div.content-container');
+    assert.equal(privacyPolicyContainer.length, 1);
+  });
+
+  describe('heading', () => {
```

Comment:
This isn't really testing what is shallow rendered but rather that the Translate component has a `h1` prop. A unit test that you could do instead would be that a `Translate` component is shallow rendered in `PrivacyPolicy`:

```
it('renders a <Translate /> component', function() {
  assert.equal(wrapper.find('Translate').length, 1);
});
```